src/yunohost/settings.py

```diff
@@ -39,6 +40,10 @@
     ("security.password.admin.strength", {"type": "int", "default": 1}),
     ("security.password.user.strength", {"type": "int", "default": 1}),
     ("service.ssh.allow_deprecated_dsa_hostkey", {"type": "bool", "default": False}),
+    ("service.ssh.ciphers.compatibility", {"type": "enum", "default": "modern",
+        "choices": ["intermediate", "modern"]}),
+    ("security.ciphers.compatibility", {"type": "enum", "default": "intermediate",
```

Comment:
Eeh can we try to call those in a symetric fashion ? I don't know what's best ... from what I understand you wanted to keep `security` because there were already `security.password.*` settings... And then there was also a `service.ssh.*` setting ... (which now that I look at it, should have also been called security because it's related to security...)

Sooo meh, idk, I would call these just 

```
security.compatibility.ssh
security.compatibility.nginx
```

?

Reply:
Did you say naming is hard ? 😄 
I think security is a major point so I would name all those settings to begin with security. Then there is a compatibility matrix which affects most of the yunohost services.
May be we need an advice from someone with a biggest picture ?

Reply:
@alexAubin Do you think we should wait for more comments ?

Reply:
I implemented what I was thinking about in https://github.com/YunoHost/yunohost/pull/640/commits/fee79820e097ebefb342bd53a16098661359b196